04-single-hop-6lowpan-icmp/test_spec04.py

```diff
@@ -18,6 +18,7 @@ class Shell(Ifconfig, GNRCICMPv6Echo, GNRCPktbufStats):
     pass
 
 
+@pytest.mark.flaky(reruns=3, reruns_delay=30)
```

Comment:
What do you think about defining some defaults for this maybe using `pytest_configure`? Feel free to dismiss though.

Reply:
If you tell me, what you mean by that I might oblige. The [doc on `pytest_configure`](https://docs.pytest.org/en/6.2.x/reference.html#pytest.hookspec.pytest_configure) isn't all that helpful.

Reply:
For posterity: @fjmolinas and I agreed out of band, that we both do not know how to use `pytest_configure` properly and that I in addition fear that hiding config there would make the code harder to understand.